tests/vm_startup_test.go

```diff
@@ -0,0 +1,162 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+const timeout = 3 * time.Minute
+const pollingInterval = 5 * time.Second
+const testNamespace = "secondary-test"
+const dnsPort = "31111"
+const dnsIP = "127.0.0.1" // Forwarded to the node port - https://github.com/kubevirt/kubevirtci/pull/867 ,https://github.com/kubevirt/kubesecondarydns/pull/23
+const domain = "vm.secondary.io"
```

Comment:
what about reading it from the CR and building accordingly the expected string?
this way it will work for:
* empty string
* D/S that fill the cluster-name (if this what we are going to do at the end)
* non empty string

Reply:
I think it is an overkill for the test. If we will ever need it on d/s, then we can change it.

Reply:
I think that this test might even help us to test it D/S even manually
but ok we can just update upon need,
what about reading it from os.getenv that allows to config it easily when running D/S ?